lib/cli.js

```diff
@@ -10,15 +10,17 @@ const parseArgs = require('./args');
   const fn = require(`./commands/${command}`)
   commist.register(command, async (args) => {
     const opts = parseArgs(args)
-    if (!needToShowHelp(command, opts)) {
-      try {
-        const out = await fn(opts)
-        if (out) {
-          logger.info(out)
-        }
-      } catch (error) {
-        logger.error(error)
+    if (needToShowHelp(command, opts)) {
+      return
+    }
+    try {
+      logger.debug('Starting %s command', command)
+      const out = await fn(opts)
+      if (out) {
+        logger.info(out.message)
       }
```

Comment:
This is problematic for draft. We should print the full draft instead.

Reply:
This was not resolved. In other term, we should not use the logger here.

Reply:
Sorry but I don't understand what you meant then 😅 I'm missing the meaning of "We should print the full draft instead."
I was thinking that you referenced the `out.message` so I put all the `out`.

So must I change the `logger` with the `console.log`? Or something else?

Reply:
passing it to the logger means that \n will appear as \n, and not as actual newlines. As a result, copy pasting this is cumbersome where it should not be.